notebook/static/notebook/js/codecell.js

```diff
@@ -160,6 +160,15 @@ define([
 
         var input = $('<div></div>').addClass('input');
         this.input = input;
+
+        var run_this_cell = $('<div></div>').addClass('run_this_cell');
+        run_this_cell.prop('title', 'Run this cell');
+        run_this_cell.append('<i class="fa-step-forward fa"></i>');
+        run_this_cell.mouseup(function (event) {
```

Comment:
Is there a reason to do this on the mouseup event rather than the click event? I think we usually use click.

Reply:
Ok, I'll replace it with click event